python/cuml/tests/test_sklearn_import_export.py

```diff
@@ -733,6 +757,27 @@ def test_random_forest_regressor(random_state):
     assert sk_model2.score(X, y) > 0.7
     assert cu_model2.score(X, y) > 0.7
 
+    # Test with OOB score computation
+    cu_model_oob = cuml.RandomForestRegressor(oob_score=True).fit(X, y)
+    sk_model_oob = sklearn.ensemble.RandomForestRegressor(oob_score=True).fit(
+        X, y
+    )
+
+    sk_model2_oob = cu_model_oob.as_sklearn()
+    cu_model2_oob = cuml.RandomForestRegressor.from_sklearn(sk_model_oob)
+
+    # Ensure params/attrs roundtrip with OOB score
+    assert_roundtrip_consistency(cu_model_oob, cu_model2_oob)
+
+    # Verify OOB attributes are present
+    assert hasattr(cu_model_oob, "oob_score_")
+    assert hasattr(cu_model2_oob, "oob_score_")
+    assert hasattr(sk_model2_oob, "oob_score_")
```

Comment:
I think that the `assert_roundtrip_consistency` does not check for equality, but in this case it should be a simple copy. Could we quickly check that the `oob_score_` isn't modified?